tasks/e2e-monorepos.sh

```diff
@@ -110,25 +110,48 @@ pushd "$temp_app_path"
 cp -r "$root_path/packages/react-scripts/fixtures/monorepos/yarn-ws" .
 cd "yarn-ws"
 cp -r "$root_path/packages/react-scripts/fixtures/monorepos/packages" .
-yarn
 
 # Test cra-app1
-cd packages/cra-app1
+pushd packages/cra-app1
+cp -r "$root_path/packages/react-scripts/fixtures/monorepos/cra-app/"* .
+yarn
+verifyBuild
+yarn start --smoke-test
+verifyTest
+popd
+
+# ******************************************************************************
+# Test clean create-react-app inside workspace
+# ******************************************************************************
+pushd packages
+npx create-react-app newapp
+cd newapp
+yarn
```

Comment:
Do we want to run yarn automatically when creating the app (ie., inside init.js) to avoid running it here?

Reply:
Not sure what you mean. We use yarn to create an app if it’s installed.

Reply:
Yeah, you're right, I think this might be due to a yarn bug not creating the react-scripts .bin link when we run "yarn add react-scripts" for the new app.  Needs some more debugging.